### PR TITLE
Add configuration for cargo-embed; switch examples to use `panic-rtt-target`

### PR DESCRIPTION
**Pull request by hannobraun**
I've left the OpenOCD configuration in the repository, as cargo-embed isn't a full replacement yet (GDB server doesn't seem to be fully-featured). But I think this configuration is a better default, especially with probe-rs's pace of development.

### CODE REVIEW

**Comment by david-sawatzke** [comment]
Hmm, doesn't quite work for me D: : 
```
$ cargo flash --version
cargo-flash 0.7.0
$ cargo flash --chip LPC845M301JHI48 --features 845-rt --example gpio_delay --release
    Finished release [optimized + debuginfo] target(s) in 0.03s
    Flashing [..]/target/thumbv6m-none-eabi/release/examples/gpio_delay
     Erasing sectors ✔ [00:00:00] [###########################]   5.00KB/  5.00KB @   5.67KB/s (eta 0s )
 Programming pages   ✔ [00:00:01] [###########################]   4.25KB/  4.25KB @   1.98KB/s (eta 0s )
$ cargo embed --example gpio_delay --features 845-rt --release
    Finished release [optimized + debuginfo] target(s) in 0.03s
    Flashing [..]/target/thumbv6m-none-eabi/release/examples/gpio_delay
        WARN probe_rs::architecture::arm::memory::romtable > Component at 0x5000c000: CIDR0 has invalid preamble (expected 0xd, got 0x0)
       Error failed attaching to target

Caused by:
    0: Unable to load specification for chip
    1: The connected chip could not automatically be determined.
$ cargo embed --version
cargo-embed 0.8.0
```

Gonna need to debug this

**Comment by hannobraun** [comment]
You're not activating the right configuration. This should work (changed part bold):
> cargo embed **lpc845** --example gpio_delay --features 845-rt --release

This is of course not obvious, unless you've immersed yourself into cargo-embed for the last weeks, and should really be documented somewhere. I'll push another commit adding this to the README tomorrow.

**Comment by david-sawatzke** [comment]
> You're not activating the right configuration

Oh, now it works. Thanks!

**Comment by hannobraun** [comment]
Pushed new commits with the documentation. Also fixed some broken links in the README.

It would be nice, if we could infer the cargo-embed configuration from the Cargo features, but I'm not tackling that right now.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,16 @@ lpc8xx-hal = "0.7"
...
 If you want to use LPC8xx HAL in an application (as opposed to a library), there are additional things that need to be set up. Please refer to the [API Reference] for details.
 
+To run one of the examples from this repository, please adapt the following command if you're using an LPC845-BRK board:
+```
+cargo embed lpc845 --example gpio_delay --features 845-rt
+```
+
+Or adapt the following command if using an LPCXpresso824-MAX board:
+```
+cargo embed lpc82c --example gpio_delay --features 82x-rt
```
Comment (david-sawatzke) [inline README.md:45]: s/lpc82c/lpc82x/

Comment (hannobraun) [reply README.md:45]: Good catch. I did find the same typo when test-executing those commands, but I guess somehow no fix made its way back.

**Comment by hannobraun** [comment]
Rebased and pushed a new commit that fixes the typo. Merging, as this was already approved.

**Pull request merged**